sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_base/view.sql

```diff
@@ -69,7 +69,7 @@ all_events AS (
     fenix
 )
 SELECT
-  `timestamp`,
+  date(`timestamp`) AS submission_date,
```

Comment:
Instead of grouping on the timestamp, we should group on the date instead to aggregate data and make querying more efficient. I've done this previously with the search metrics dataset, now that we also have Fenix data the enrollment queries fail to run due to memory exceeded errors